manifests/params.pp

```diff
@@ -6,21 +6,23 @@
 #
 class puppetboard::params {
 
-  # puppetlabs/apache names the apache service
-  # resource 'httpd', nevermind the actual os
-  $apache_service = 'httpd'
-
   case $::osfamily {
```

Comment:
please use the facts hash here

Reply:
I did see those, and had left them the same as the original PR. I added an additional commit that should switch to structured facts. 

Acceptance tests are another story; they were out of date, so I'm working on another pull to at least somewhat fix them.